contrib/cirrus/mac_setup.sh

```diff
@@ -31,3 +31,10 @@ echo "TMPDIR=/private/tmp/ci" >> $CIRRUS_ENV
 
 # Removed completely during cleanup.
 mkdir -p /private/tmp/ci
+
+# Some test operations & checks require a git "identity"
+_gc="git config --file $HOME/ci/.gitconfig"
+$_gc user.email "TMcTestFace@example.com"
+$_gc user.name "Testy McTestface"
+# Bypass git safety/security checks when operating in a throwaway environment
+git config --global --add safe.directory $CIRRUS_WORKING_DIR
```

Comment:
If this should be `$_gc` (to set the HOME/ci one), please change it

If this is correct as it is, please add a comment explaining that it's deliberate and explaining why

Reply:
Nope, it was a half-blind copy-paste, but the original used `--system`.  I can update it to use `$_gc`, thanks for the tip.